src/util/thread.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "util_error.h"
+
+#include <windef.h>
+#include <winbase.h>
+
+namespace dxvk {
+  class thread {
```

Comment:
Can you add a small comment explaining why this class is needed? It's obvious from the commit message, but it might not be obvious when reading the code that this is needed for winelib builds.

Reply:
Right, that sounds useful, I will add it.